spec/fixtures/zoom/meeting_details_with_zero_participants.json

```diff
@@ -0,0 +1,16 @@
+{
+  "uuid": "X9GT4HFTSSahMWhD8ErjYg==",
+  "id": 96428502996,
+  "host_id": "VTOR3RckQRSDd5OeKkOfkQ",
+  "type": 2,
+  "topic": "ReadMe Workshop",
+  "user_name": "Meg Stang (she/her)",
+  "user_email": "mstang@turing.io",
+  "start_time": "2023-01-10T15:45:22Z",
+  "end_time": "2023-01-10T17:00:03Z",
+  "duration": 75,
+  "total_minutes": 1724,
+  "participants_count": 0,
```

Comment:
We can only create fixtures for responses we could potentially receive from the api. I don't see any where in the [zoom api docs](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/meeting) for the field participants_count, so we can't expect to receive it.

Reply:
Removed fixture